### PR TITLE
clarify @safe

### PR DESCRIPTION
**Pull request by wilzbach**

### CODE REVIEW

File: public/content/en/basics/memory.md [hunk 1]
```diff
@@ -30,7 +30,8 @@ code that is marked as @safe.
     }
 
 Unless specified otherwise the default is `@system`.
-Using `@safe`, a subset of D can be forced to prevent memory bugs by design.
+Using `@safe`, a subset of D, many memory bugs can be
```
Comment (stonemaster) [inline public/content/en/basics/memory.md]: SafeD is a subset of D. This sentence sounds a little bit misleading to me..

Comment (wilzbach) [inline public/content/en/basics/memory.md]: Hmm both variants are confusing :/

How about:

> D has a `@safe` subset that prevents memory bugs by design

**Comment by stonemaster** [comment]
That sounds great and in the point! :+1:

**Comment by wilzbach** [comment]
> That sounds great and in the point! :+1:

Rebased & merged :)

**Pull request merged**